docs/ADDING_NUCLEAR_FUELS.md

```diff
@@ -0,0 +1,185 @@
+# Adding nuclear fuels
+To add fission fuel rods and control rods, you will need to use a startup script and the events in `MIMaterialEvents`
+Fuel rods and control rods are parts that can be added to a new or preexisting material.
+
+Isotope Fuel Parameters and Isotope Parameters can also be added. They need a startup script and the events in `NuclearConstantEvents`
+
+## Add a fuel rod
+Here is an example script that adds a new `LE Americium MOX` material that has fuel rods.
+For a full tutorial on how to add new materials, refer to `ADDING_MATERIALS.md`
+
+```javascript
+MIMaterialEvents.addMaterials(event => {
+
+    event.createMaterial('LE Americium MOX', 'le_americium_mox', 0x83867B,
+    builder => {
+        // Every material with a fuel rod needs at least the ingot and rod parts
+        builder.addParts('ingot', 'rod') 
+               .fuelRod(
+                    0.6,     // Thermal absorption probabilty
+                    0.35,    // Thermal scattering
+                    2400,    // Max temperature
+                    900,     // Temperature limit (high)
+                    2300,    // Temperature limit (low)
+                    8,       // Neutron multiplication
+                    0.5),    // Direct energy factor
+               .defaultRecipes();
+    });
+    
+});
+```
+
+You can also mix two different isotope fuel parameters. This can be used to simulate MOX and MOX-type fuels.
+The resulting parameters will be `r * a + (1 - b) * b`, where `r` is the factor, `a` is the first set of parameters and `b` is the second 
+The example below illustrates how a fuel rod can be created from the U238 and Pu239 parameters
+This generates the same parameters as LE MOX fuel
+
+``` javascript
+[...]
```

Comment:
You should provide full examples to reduce the amount of questions by people not familiar with JS.